src/Uno.SampleDependency/Class1.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Uno.SampleDependency
 {
-	public class Class1
+	public class MyClass
```

Comment:
Is this still used ?

Reply:
Probably in the samples, not sure

Reply:
its odd the class name isn't the same as the file name

Reply:
@jeromelaban ill merge into beta, but fyi see above comments!